use_cases/eluc/prescriptors/nsga2/train_prescriptors.py

```diff
@@ -1,32 +1,46 @@
+"""
+Script used to train NSGA-II prescriptors.
+Requires a config file with the same fields as shown in the
+test.json file in prescriptors/nsga2/configs
+"""
+
+import argparse
+import json
 from pathlib import Path
 
-from data import constants
 from data.eluc_data import ELUCData
 from prescriptors.nsga2.torch_prescriptor import TorchPrescriptor
 from predictors.neural_network.neural_net_predictor import NeuralNetPredictor
 
 if __name__ == "__main__":
+
+    # Load config
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--config_path", type=str, required=True)
+    args = parser.parse_args()
+    with open(Path(args.config_path), "r", encoding="utf-8") as f:
+        config = json.load(f)
+
     print("Loading dataset...")
     dataset = ELUCData()
+
     print("Loading predictor...")
+    # TODO: We need to make it so you can load any predictor here
     nnp = NeuralNetPredictor()
-    nnp.load("predictors/neural_network/trained_models/no_overlap_nn")
+    nnp_path = Path(config["predictor_path"])
+    nnp.load(nnp_path)
+
     print("Initializing prescription...")
-    candidate_params = {"in_size": len(constants.CAO_MAPPING["context"]),
-                        "hidden_size": 16,
-                        "out_size": len(constants.RECO_COLS)}
+    if "seed_dir" in config["evolution_params"].keys():
+        config["evolution_params"]["seed_dir"] = Path(config["evolution_params"]["seed_dir"])
```

Comment:
Convert seed dir string into Path object here so that we don't have to do it inside the TorchPrescriptor object and can continue assuming that what gets passed in is a Path